### PR TITLE
photo and status stuff 

### LINKED ISSUE #61: Provide proof of work
A worker should be able to start works, and end them. 
A work event should have 4 statuses:
- awaiting
- started
- proofawait
- finished

Provide an endpoint to modify the state also to accept one or multiple pictures and assign them as proof of work.

### PR DESCRIPTION
**Pull request by akosgdcsi**
i dont know if its works for you guys, because gitkraken doin some  interesting stuff

### CODE REVIEW

File: wman-backend/Wman.Logic/Classes/EventLogic.cs [hunk 1]
```diff
@@ -351,6 +351,62 @@ public async Task DnDEvent(int Id, DnDEventDTO newWorkEvent)
...
+
+            if (Status.finished == workevent.Status)
+            {
+                throw new InvalidOperationException(WmanError.StatusFinished);
+            }
+            if (Status.proofawait == workevent.Status && workevent.ProofOfWorkPic.Count == 0)
+            {
+                throw new InvalidOperationException(WmanError.StatusPowMissing);
+            }
+            // TODO: Innentől átadni.
```
Comment (tsaskater) [inline wman-backend/Wman.Logic/Classes/EventLogic.cs]: Please remove this unnecessary  TODO as it's not a todo.
Otherwise good job. Works as intended.

**Review by tsaskater — CHANGES_REQUESTED** [review]
Remove the todo pls... 😠🙅🏿‍♀️
Otherwise I love it.

**Review by tsaskater — APPROVED** [review]
You haven’t missed a thing.

File: wman-backend/Wman.Logic/Classes/EventLogic.cs [hunk 2]
```diff
@@ -351,6 +351,57 @@ public async Task DnDEvent(int Id, DnDEventDTO newWorkEvent)
...
+                case Status.started:
+                    break;
+                case Status.proofawait:
+                    workevent.WorkFinishDate = DateTime.Now;
+                    break;
+                case Status.finished:
+                    break;
+                default:
+                    break;
+            }
```
Comment (pop-simon) [inline wman-backend/Wman.Logic/Classes/EventLogic.cs:381]: Do not use both `if-else` and `switch-case` though you can use an `if` in a `case`. If you choose `switch-case` the default case should throw an `Exception` like `Status not found`.

**Review by pop-simon — CHANGES_REQUESTED** [review]
Well done, but small changes required 👌

**Comment by akosgdcsi** [comment]
If somehow the code gets to the default case, 10000 puppies shall rot in hell

**Comment by akosgdcsi** [comment]
kép

**Review by pop-simon — APPROVED** [review]
That is not what I meant but nevermind

**Pull request merged**